resources/shaders/rt_reflection_shadows.hlsl

```diff
@@ -146,7 +146,7 @@ float3 DoReflection(float3 wpos, float3 V, float3 normal, uint rand_seed, uint d
 #define M_PI 3.14159265358979
 
 [shader("raygeneration")]
-void RaygenEntry()
+void HybridRaygenEntry()
```

Comment:
Hybrid is not a good anymore.

Reply:
It's there for compatibility reasons since some parts of the engine still use it. Emilio's shader refactoring takes care of this.